Maskbook/Scene/App/RedPacket/View/NFTRedPackedConfirmView.swift

```diff
@@ -0,0 +1,126 @@
+    @InjectedProvider(\.mainCoordinator) private var mainCoordinator
+   
+    var tipsView: some View {
+        HStack(spacing: 10) {
+            Asset.Images.Scene.Social.connectHintBannerIcon.asImage()
+                .resizable()
+                .aspectRatio(contentMode: .fit)
+                .frame(width: 24)
+            Text(L10n.Plugins.Luckydrop.Confirm.tips)
+                .font(FontStyles.rh6.font)
+                .foregroundColor(Asset.Colors.Public.info.asColor())
+        }
+        .frame(maxWidth: .infinity)
+        .padding(.horizontal, 16)
+        .padding(.vertical, 10)
+        .background(Asset.Colors.Public.infoBg.asColor())
+        .cornerRadius(12)
+    }
+    
+    var body: some View {
+        VStack(spacing: 20) {
+            Text(L10n.Plugins.Luckydrop.Confirm.title)
+                .font(FontStyles.bh4.font)
+                .foregroundColor(Asset.Colors.Text.dark.asColor())
+            VStack(spacing: 16) {
+                buildRow(
+                    title: L10n.Plugins.Luckydrop.Confirm.walletAccount,
+                    value: .plain(viewModel.address)
+                )
+                buildRow(
+                    title: L10n.Plugins.Luckydrop.Confirm.attachedMessage,
+                    value: .plain(viewModel.message)
+                )
+                buildRow(
+                    title: L10n.Plugins.Luckydrop.Confirm.totalAmount,
+                    value: .plain("\(viewModel.totalAmount)")
+                )
+                buildRow(title: L10n.Plugins.Luckydrop.Confirm.transactionFee, value: .gas(viewModel.gasFeeInfo))
+                tipsView
+            }
+            PrimaryButton(
+                title: L10n.Common.Controls.confirm,
+                animating: viewModel.isSending,
+                isEnable: !viewModel.isSending
+            ) {
+                viewModel.onConfirm()
+            }
+        }
+        .padding(.horizontal)
+    }
+    
+    @ViewBuilder
+    func buildRow(title: String, value: ValueType) -> some View {
+        HStack {
+            Text(title)
+                .font(FontStyles.rh6.font)
+                .foregroundColor(Asset.Colors.Text.dark.asColor())
+            Spacer()
+            if case .plain(let title) = value {
+                Text(title)
+                    .truncationMode(.middle)
+                    .font(FontStyles.bh6.font)
+                    .foregroundColor(Asset.Colors.Text.dark.asColor())
+                    .frame(maxWidth: 118)
+                    .fixedSize()
+            } else if case .token(let tokenURL, let value) = value {
+                HStack(spacing: 4) {
+                    KFImage(tokenURL)
+                        .cancelOnDisappear(true)
+                        .loadImmediately()
+                        .resizable()
+                        .aspectRatio(contentMode: .fit)
+                        .frame(width: 24)
+                        .cornerRadius(12)
+                    Text(value)
```

Comment:
Highly recommend leaving a space between `View`s when they are in the same scope.